cluster/gce/gci/configure-helper.sh

```diff
@@ -921,168 +921,6 @@ EOF
   fi
 }
 
-# Write the config for the audit policy.
```

Comment:
I understand your motivation of removing it from here.

That said, the fact that it lives here means that we test that as part of OSS scalability tests (and audit has non-negligible impact on performance/scalability). I can see a lot of value in its continuous testing at scale.

If we want to get rid of that, I would like to see some kind-of replacement that will mitigate the probability of regression that will appear to be hard to debug/fix once we observe it much later in GKE.

/hold

Reply:
How are scalability tests initiated?

Reply:
The cluster is created the same way as for all e2e tests (via prow)
Example config is here: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml#L3

Reply:
Thanks for explaining.

Reply:
Thanks for explaining.